fitlins/interfaces/bids.py

```diff
@@ -202,13 +204,11 @@ def _run_interface(self, runtime):
             info['confounds'] = confounds_file
             info['repetition_time'] = TR
 
-            # Transpose so each contrast gets a row of data instead of column
-            contrasts = block.get_contrasts([contrast['name']
-                                             for contrast in block.contrasts],
+            # Make contrast for each HRF_variable
+            contrasts = block.get_contrasts([con for con in block.model['HRF_variables']],
                                             **ents)[0][0].T
             # Add test indicator column
-            contrasts['type'] = [contrast['type']
-                                 for contrast in block.contrasts]
+            contrasts['type'] = ['T' for contrast in contrasts]
```

Comment:
See [this section](https://github.com/poldracklab/fitlins/pull/30/files#diff-4366269db32fec3a5668894ac5e8ee19R258) of #30 for how I'm planning to handle identity contrasts.

Reply:
Also a temporary hack.

Reply:
I don't think we need to handle identity contrasts any differently from other contrasts... In theory the changes @adelavega made should handle any kind of contrast. The key to getting identity contrasts working with nistats is to drop all the rows that have weights of `0`.